index.css

```diff
@@ -1,20 +1,77 @@
 :root{
-    --backgroundColor: #F8F4FA;
-    --DarkPurple : #856595;
+    --backgroundColor: #FAFAFA;
+    --DarkPurple : #825F96;
+    --lightText: #C0AEC9;
+    --midPurple: #997DA9;
 }
 
-section{
+*{
+    font-family: 'Poppins', sans-serif;
+    color: var(--DarkPurple);
+}
+
+html{
+    font-size: 1.5vw;
+}
+
+body{
     background-color: var(--backgroundColor);
-    /* position: relative; */
-    /* width: 100%;
-    height: 100%; */
-}
-
-.personalIMG{
-    display: inline-block;
-    /* position: relative; */
-    /* width: 20px;
-    height: 30px; */
-    background-color: var(--DarkPurple);
-    
-}
\ No newline at end of file
+    color: var(--DarkPurple);
+}
+
+
+header{
+    position: fixed;
+    width: 100%;
+    height: 7%;
+    display: flex;
+    justify-content: right;
+}
+
+nav {
+    position: absolute;
```

Comment:
why? no need.

Reply:
because It was skewed to the right and I had to use the right property, but now I removed it and solved it using the justify-content